sdks/java/io/cdap/src/main/java/org/apache/beam/sdk/io/cdap/MappingUtils.java

```diff
@@ -49,6 +52,6 @@ public class MappingUtils {
       return Plugin.create(
           pluginClass, ServiceNowInputFormat.class, SourceInputFormatProvider.class);
     }
-    return null;
+    throw new UnsupportedOperationException("Given plugin class is not supported!");
```

Comment:
we should have it include what class it got in the exception message, to make debugging easier

Reply:
Done